### ISSUE TITLE
Title of minimized suspended tabs isn't displayed upon Chrome restart

### ISSUE THREAD

**Issue by valerieosmith** [deanoemcke/thegreatsuspender#466]
When restarting Chrome, if a suspended tab is minimized, the title in the Window menu is "Suspended Tab" until the tab is reloaded. Version 6.21

**Comment by deanoemcke** [comment]
Could you please describe a set of exact steps to reproduce this issue? 
I'm not sure exactly what you mean by a 'minimized tab' and where you are viewing the 'window menu title'.
Also, can you test this on the latest version (v6.30)?
Thanks.

**Comment by valerieosmith** [comment]
@deanoemcke This is on MacOS Sierra (10.12.6). 

Repro steps:
1) Open at least two windows. Load a page in each one. 
2) Suspend a tab in one of those windows. Minimize the window (Command + M) while that suspended tab still has the focus.
3) Quit Chrome
4) Start Chrome

Observe that the minimized window (accessible via the Window menu on the top of the screen) shows the title of the minimized window as "Suspended Tab" as opposed to the title of the page once it is loaded. Once the tab is unsuspended, the title is updated and displayed correctly.

**Comment by deanoemcke** [comment]
I managed to replicate this once, but then not again.
I'm honestly not sure what is going on. Does it make a difference when you click on the window menu? I found that if you click on it very early, it will show the url. But then reclicking on the window menu will subsequently show the correct tab title.

**Comment by valerieosmith** [comment]
@deanoemcke I can repro this 100% of the time. I can take a video if that would help. It does not seem to make a difference if I click on the window menu. Even after I restore a minimized window (whose tabs are all suspended, including the foreground tab), that window's title in the Window menu will still read "Suspended Tab"

**Comment by valerieosmith** [comment]
@deanoemcke Here's a video of the issue: https://youtu.be/ULxFL6SGK2g

**Comment by deanoemcke** [comment]
Thanks a lot for the video. I've managed to recreate it again. I'm getting this error in the console of the offending page:
```
Uncaught TypeError: Cannot read property 'getSuspendedTitle' of undefined
    at suspended.js:4
```
I think this is essentially the same issue as: https://github.com/deanoemcke/thegreatsuspender/issues/597

I guess that minimized chrome windows exacerbate the problem of tabs loading before the extension is ready. 

In the upcoming release, I have actually added some fallback code that will allow the page to suspend properly even if it cannot make contact with the extension (https://github.com/deanoemcke/thegreatsuspender/issues/591).

This should resolve the issue. Thanks for taking the time to help me debug this.
